apps/studio/src/components/PageEditor/Editor.tsx

```diff
@@ -61,32 +62,36 @@ const renderers: JsonFormsRendererRegistryEntry[] = [
 
 const Editor = ({ jsonSchema, editorValue, onChange }: any) => {
   return (
-    <Box>
-      <ThemeProvider theme={jsonFormsTheme}>
-        <CssBaseline />
-        <Box
-          padding="0.5rem"
-          overflowY={'auto'}
-          maxH="full"
-          css={{
-            '& div': {
-              maxWidth: '100% !important', // Setting max-w-full with high specificity
-            },
-            '& div.MuiTabs-flexContainer': {
-              overflowX: 'auto',
-            },
-          }}
-        >
-          <JsonForms
-            schema={jsonSchema}
-            data={editorValue}
-            renderers={renderers}
-            cells={materialCells}
-            onChange={({ data }) => onChange(data)}
-          />
-        </Box>
-      </ThemeProvider>
-    </Box>
+    <ComponentSelector
+      onClose={() => console.log('close')}
```

Comment:
nit: toast is clearer for an indication that `onClose` is firing

Reply:
Leaving this for now since we'll be replacing it shortly!